repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tasks/ClassificationTask.java

```diff
@@ -180,5 +184,9 @@ protected void setStatus(AtlasTask.Status status) {
         graph.commit();
     }
 
+    protected void setAssetsCountToPropagate(Long assetsCount) {
```

Comment:
Let's remove this code which was used for debugging.

Reply:
Removed, thanks for pointing out